### PR TITLE
Revert "[test] Trigger auto-roll CL"

### PR DESCRIPTION
**Pull request by CaseyHillers**
Reverts flutter/cocoon#1299 to test out the auto-roll CL again

### CODE REVIEW

**Review by christopherfujino — APPROVED** [review]
LGTM

**Comment by CaseyHillers** [comment]
Waiting on https://flutter-review.googlesource.com/c/recipes/+/16123 to land + propagate before merging

**Pull request merged**
